test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,22 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNullTest() {
```

Comment:
Recall from tutorial if lumping all test conditions within a Junit test is better or separating the Junit test by test condition is a better?